src/status_im/constants.cljs

```diff
@@ -200,6 +200,7 @@
 (def ^:const web3-send-transaction "eth_sendTransaction")
 (def ^:const web3-personal-sign "personal_sign")
 (def ^:const web3-get-logs "eth_getLogs")
+(def ^:const web3-sign "eth_sign")
```

Comment:
eth_sign is deprecated, please use web3-personal-sign instead

Reply:
now I get 

```
:code 32700
:message "method is unsupported by RPC interface"
```

Reply:
sorry, i edited my comment